Tools/machines/pitzer-osc/install_dependencies.sh

```diff
@@ -0,0 +1,121 @@
+
```

Comment:
Do you want to add a short block here, e.g., as we do on Perlmutter, to check if `$proj` is set?

Since you start here with `rm -rf ${SW_DIR}` I would set `$SW_DIR` here again OR better check that the profile script did set it and if not error out.